src/Uno.UWP/Generated/3.0.0.0/Windows.ApplicationModel.Contacts/ContactBatch.cs

```diff
@@ -2,12 +2,12 @@
 #pragma warning disable 114 // new keyword hiding
 namespace Windows.ApplicationModel.Contacts
 {
-	#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
+	#if false || __IOS__ || NET461 || __WASM__ || __MACOS__
```

Comment:
If the class does not contain platform specific code, make it available for all platorms.

Reply:
But I don't know if it contain platform specific code :)
For Android implementation, I added constructor:
`internal ContactBatch(IReadOnlyList<Contact> contacts)`
i don't know what iOS implementation would be, so I don't know if it will be used, or what will be implemented.

Also, whole Reader tree is only for Android, and Contact as such can be used also in other scenario. Maybe making ContactBatch 'implemented' for e.g. iOS would be misleading for someone who want to search contacts on other (than Android) platforms?
I don't know, so I ask.

Reply:
No worries about asking that's fine :) The point of my comment is that even if the class is not part of an implemented set for the platform, since an instance can only be used through an implemented API, it does not matter much.

If the code you're adding is effectively cross-platform because it does not contain any platform specific code, it's fine if it's considered implemented, even if it cannot be used because the associated APIs are not.